must-gather/collection-scripts/gather

```diff
@@ -10,7 +10,7 @@ resources+=(nodes machineconfigs machineconfigpools featuregates kubeletconfigs)
 
 # run the collection of resources using must-gather
 for resource in ${resources[@]}; do
-  /usr/bin/oc adm inspect --dest-dir must-gather --all-namespaces ${resource}
+  oc adm inspect --dest-dir must-gather --all-namespaces ${resource}
```

Comment:
please keep the abspath (may need some amends)

Reply:
oc is no longer explicitly copied because it exists in the new CLI base

Reply:
we can but might cause issues d/s. why do you think we should keep it? 
A midstream MR depends on this actually, since there we use another dockerfile in terms of the base we aim to no longer copy the oc binary. I still need to see how m/s will end up. we can use an env var for the binary if that helps

Reply:
which kinds of issue you see here? I don't see the benefit in relaxing the path requirements here. Since we control (and we use a base image which we control) the base image, we know where `oc` must be, so we can use a absolute path

Reply:
I haven't got the chance to check this yet, but using the m/s build was shouting that we don't have the oc binary installed in this path. I need to revisit the m/s batch and handle installing openshift clients properly first.

Reply:
dropped this change we don't need it coupled with the base-image update